### ISSUE TITLE
Merge improvements into upstream project?

### ISSUE THREAD

**Issue by akloeckner** [onlyoneme/home-assistant-custom-components-wiim#10]
I wonder whether the improvements in this fork could be merged back into the upstream repository with @nagyrobi.  I have started by adding basic support for Wiim devices, see https://github.com/nagyrobi/home-assistant-custom-components-linkplay/pull/110. But as @onlyoneme points out, this leaves a few open questions. (Which I am not sure, they might even apply to Arylic devices, too.) I figure, it would be more efficient, if we concentrated our work on one common project. Also, this would deliver the maximum number of features to both halves of the LinkPlay world (Arylic and Wiim), e.g. multi-room + UPnP interfacing. What do you think?

Related #7, #6 .

**Comment by onlyoneme** [comment]
Probably you're right but the fact is that I'm not a python developer. I have only the most basic knowledge of this language and I don't think I could ever merge it. Some changes are quite substantial, for example UPnP only constant polling instead of http api because the Mini loses its stability when being queried with http api all the time.

**Comment by akloeckner** [comment]
Yeah, I noticed you did quite some work. And I understand that merging this back might not be as straight forward as I'd hoped. However, as I use a mixture of WiiM and other LinkPlay devices and I really need the multi-room functionality, I will try to improve the upstream component and fix some of its flaws over time.

Would it be ok, if I tag you on WiiM related issues and ask you for advice?

**Comment by onlyoneme** [comment]
Sure, I'll try to help whenever I can.

**Comment by akloeckner** [comment]
👍 Thanks!
